application/src/main/java/com/sanction/lightning/resources/FacebookResource.java

```diff
@@ -64,4 +67,127 @@ public Response getUser(@Auth Key key, @QueryParam("username") String username)
 
     return Response.ok(facebookUser).build();
   }
+
+  /**
+   * Fetches all the photos of a specific user.
+   * @param key The authentication key for the requesting application.
+   * @param username The username of the StormUser to get FacebookUser information for.
+   * @return Array of photos
+   */
+  @GET
+  @Path("/photos")
+  public Response getPhotos(@Auth Key key, @QueryParam("username") String username)
+          throws IOException, MalformedURLException {
+    if (username == null) {
+      return Response.status(Response.Status.BAD_REQUEST)
+              .entity("'username' query parameter is required for getUser").build();
+    }
+
+    PilotUser pilotUser = thunderClient.getUser(username);
+    FacebookProvider facebookProvider
+            = facebookProviderFactory.newFacebookProvider(pilotUser.getFacebookAccessToken());
+
+    List<String> urlList;
+    try {
+      urlList = facebookProvider.getFacebookUserPhotos();
+    } catch (FacebookOAuthException e) {
+      LOG.error("Bad Facebook OAuth token for username {}", username, e);
+      return Response.status(Response.Status.NOT_FOUND)
+              .entity("Request rejected due to bad OAuth token").build();
+    }
+
+    return Response.ok(urlList).build();
+  }
+
+  /**
+   * Fetches all the photos of a specific user.
+   * @param key The authentication key for the requesting application.
+   * @param username The username of the StormUser to get FacebookUser information for.
+   * @return Array of photos
```

Comment:
This javadoc is wrong - the method is for videos not photos. Also, it should be PilotUser and not StormUser